mxd/README.md

```diff
@@ -974,7 +974,216 @@ That's because, Trudy had created the [access policy](#762-create-access-policy)
 Can you create another access policy and contract definition which allows Bob to view Trudy's catalog?  
 `Hint:` Create an access policy using Bob's BPN Number (`BPNL000000000002`) and use this access policy id while creating contract definition.
 
-## 8. Improving the setup
+## 8. S3 to S3 Transfer
+
+After the terraform command has successfully completed minio console will look like below.
+
+![img.png](assets/dashboard.png)
+
+![img.png](assets/alice_bucket.png)
+
+Create asset metadata for provider connector i.e Alice.
+
+#### 8.1 Create Asset
+Create an asset using the following `curl` command:
+```shell
+curl --location 'http://localhost/alice/management/v2/assets' \
+--header 'Content-Type: application/json' \
+--header 'X-Api-Key: password' \
+--data-raw '{
+    "@context": {},
+    "asset": {
+        "@type": "Asset",
+        "@id": "3",
+        "properties": {
+            "description": "Product EDC Demo Asset 3- S3"
+        }
+    },
+    "properties": {
+        "version": "1.0",
+        "type": "AmazonS3",
+        "name": "document_alice",
+        "id": "document_alice",
+        "contenttype": "text/plain"
+    },
+    "dataAddress": {
+        "@type": "DataAddress",
+        "secretAccessKey": "qwerty123",
+        "accessKeyId": "qwerty123",
+        "endpointOverride": "http://10.96.103.86:9000",
+        "type": "AmazonS3",
+        "blobname": "document.txt",
+        "region":"us-east-1",
+        "bucketName": "alice-bucket",
+        "keyName":"document.txt"
+    }
+}'
+```
+#### 8.2 Create Access Policy
+Create an access policy using the following `curl` command:
+```shell
+curl --location 'http://localhost/alice/management/v2/policydefinitions' \
+--header 'Content-Type: application/json' \
+--header 'X-Api-Key: password' \
+--data-raw '{
+  "@context": {
+    "odrl": "http://www.w3.org/ns/odrl/2/"
+  },
+  "@type": "PolicyDefinitionRequestDto",
+  "@id": "2",
+  "policy": {
+    "@type": "Policy",
+    "odrl:permission": [
+      {
+        "odrl:action": "USE",
+        "odrl:constraint": {
+          "@type": "LogicalConstraint",
+          "odrl:or": [
+            {
+              "@type": "Constraint",
+              "odrl:leftOperand": "BusinessPartnerNumber",
+              "odrl:operator": {
+                "@id": "odrl:eq"
+              },
+              "odrl:rightOperand": "BPNL000000000002"
+            }
+          ]
+        }
+      }
+    ]
+  }
+}'
+```
+Alice defines an access policy which allows only Bob (with BPN Number BPNL000000000002) to view its catalog.
+
+#### 8.3 Create Contract Definition
+Create a contract definition using the following `curl` command:
+```shell
+curl --location 'http://localhost/alice/management/v2/contractdefinitions' \
+--header 'Content-Type: application/json' \
+--header 'X-Api-Key: password' \
+--data-raw '{
+    "@context": {},
+    "@id": "2",
```

Comment:
St this point contract definition with `id: 2` already exist. Use another one.

Reply:
Id 3 Used